policyengine_us/variables/gov/states/mt/tax/income/subtractions/mt_disability_income_subtraction.py

```diff
@@ -0,0 +1,35 @@
+from policyengine_us.model_api import *
+
+
+class mt_disability_income_subtraction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Montana disability income subtraction"
+    defined_for = StateCode.MT
+    unit = USD
+    definition_period = YEAR
+    reference = "https://rules.mt.gov/gateway/RuleNo.asp?RN=42%2E15%2E217"
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        p = parameters(
+            period
+        ).gov.states.mt.tax.income.subtractions.disability_income
+        age_eligible = person("age", period) < p.age_threshold
+        is_head = person("is_tax_unit_head", period)
+        age_eligible_head = age_eligible & is_head
+        is_spouse = person("is_tax_unit_spouse", period)
```

Comment:
```suggestion
        age_eligible = person("age", period) < p.age_threshold
        is_head = person("is_tax_unit_head", period)
        age_eligible_head = age_eligible & is_head
        is_spouse = person("is_tax_unit_spouse", period)
```
we can use age_head and age_spouse for the calculation